tools/explorer/tt_adapter/src/tt_adapter/main.py

```diff
@@ -70,9 +74,23 @@ def execute(
             memory_layout_analysis_enabled = False
             memory_layout_analysis_policy = None
 
-        ttnn_ir = self.model_runner.run(
+        self.model_runner.run(
             model_path, memory_layout_analysis_enabled, memory_layout_analysis_policy
         )
 
         # TODO(odjuricic, #933) Parse TTNN IR and return the post optimized graph.
         return {"graphs": []}
```

Comment:
Just to confirm about it here: this is the expected return in case the execution started, if something goes wrong, it should throw an error, right?

Reply:
Yes, it is okay, I just wanted to confirm as my previous implementation was expecting some response.

I made a change to accept `graphs` as an empty array as the success response :)